tests/acceptance/features/apiGraph/fullSearch.feature

```diff
@@ -51,3 +51,23 @@ Feature: full text search
       | dav-path-version |
       | old              |
       | new              |
+
+
+  Scenario Outline: search files using a deleted tag
+    Given using <dav-path-version> DAV path
+    And user "Alice" has uploaded file with content "hello world" to "file1.txt"
+    And user "Alice" has uploaded file with content "Namaste nepal" to "file2.txt"
```

Comment:
What's the point of creating this second file? Do we really need this?

Reply:
Not needed.

Reply:
> What's the point of creating this second file? Do we really need this?
Yes, I will remove the step since there was no use in creating it.